Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -221,12 +230,14 @@ public void centerJunctionTop(GrabberCameraPipeline pipeline) {
         double xOffset, yOffset;
 
         do {
-            xOffset = pipeline.xPosition - Constants.CAMERA_CENTER_X;
-            yOffset = Constants.CAMERA_CENTER_Y - pipeline.yPosition;
+            xOffset = pipeline.xPosition-Constants.CAMERA_CENTER_X;
+            yOffset = Constants.CAMERA_CENTER_Y-pipeline.yPosition;
```

Comment:
This was easier to read with the spaces.
```suggestion
            xOffset = pipeline.xPosition - Constants.CAMERA_CENTER_X;
            yOffset = Constants.CAMERA_CENTER_Y - pipeline.yPosition;
```

Reply:
Added

Reply:
This also got set back to what it was before